runtime/near-wallet-contract/implementation/wallet-contract/src/lib.rs

```diff
@@ -506,7 +506,7 @@ fn create_ban_relayer_promise(current_account_id: AccountId) -> Promise {
         "ban_relayer".into(),
         Vec::new(),
         NearToken::from_yoctonear(0),
-        Gas::from_tgas(1),
+        Gas::from_tera(1),
```

Comment:
This is `near_sdk::Gas` (re-export from near-gas), which uses `from_tgas`, so the changes in this file must be reverted.